scripts/vpn-config.pl

```diff
@@ -1023,13 +1027,16 @@
                         vpn_die(["vpn","ipsec","site-to-site","peer",$peer,"authentication"],"$vpn_cfg_err No remote key configured for rsa key name ".
                                  "\"$rsa_key_name\" that is specified for peer \"$peer\".\n");
                     } else {
-                        $genout .= "\trightrsasigkey=\"$remote_key\"\n";
+                        if (!defined($public_keys{$rsa_key_name})) {
+                            $public_keys{$rsa_key_name} = $remote_key;
+                            $genout .= "\trightsigkey=$rsa_key_name.pub\n";
+                        }
                     }
                 }
                 # Prevent duplicate includes for rsa keys.
                 if (!defined($key_file_list{$local_key_file})) {
                     $key_file_list{$local_key_file} = 1;
-                    $genout_secrets .= "include $local_key_file\n";
+                    $genout_secrets .= ": RSA $local_key_file\n";
```

Comment:
Agreed; I'm already using `leftsigkey` and `rightsigkey` instead of `leftrsasigkey` and `rightrsasigkey`, which in strongSwan 5.2.2 are synonymous, even when using ECDSA keys. I'll leave the `authby=rsasig` as it is for now since this current branch/PR is to bring the strongSwan 5.2.2 implementation to feature parity with the existing 4.5.2 implementation. ECDSA support should be addressed in a separate branch/PR since it is a new feature.